integration/integration_test.py

```diff
@@ -0,0 +1,48 @@
+"""@package integration_test.py
+Entry point for our integration testing
+"""
+
+import logging
+import os
+import sys
+import unittest
+from integration_test_base import IntegrationTestBase
+
+import test_integration_basics
+
+
+def determineIpVersionsFromEnvironment():
+  ipv4 = True
```

Comment:
I think you might want to model this as a `namedtuple`

Reply:
Actually, thinking about it, if you just have IPV4, IPv6 as enum-like constants and return a list here of the actual versions that are supported, it simplifies logic elsewhere.

Reply:
Fixed, PTAL